src/common/BaseAnnotator.ts

```diff
@@ -66,6 +66,7 @@ export default class BaseAnnotator {
         // Add custom handlers for events triggered by the Preview SDK
         this.addListener(LegacyEvent.SCALE, this.handleScale);
         this.addListener(Event.ACTIVE_SET, this.setActiveAnnotationId);
+        this.addListener(Event.ANNOTATION_DELETE, this.deleteAnnotation);
```

Comment:
Should we name these something like `ANNOTATION_REMOVE` rather than `DELETE`, since the latter suggests async CRUD behavior?